openrewrite/src/javascript/printer.ts

```diff
@@ -0,0 +1,2010 @@
+import * as J from '../java';
+
+import {
+    JContainer,
+    JLeftPadded,
+    JRightPadded,
+    Literal,
+    Semicolon,
+    Space,
+    TrailingComma
+} from '../java';
+import * as JS from '.';
+import {
+    Cursor,
+    Markers,
+    PrintOutputCapture,
+    Tree
+} from "../core";
+import {JavaScriptVisitor, JsContainer, JsLeftPadded, JsRightPadded, JsSpace, NamespaceDeclaration} from ".";
+
+export class JavaScriptPrinter<P> extends JavaScriptVisitor<PrintOutputCapture<P>> {
+
+    JAVA_SCRIPT_MARKER_WRAPPER: (out: string) => string = (out) => `/*~~${out}${out.length === 0 ? "" : "~~"}>*/`;
+
+    constructor(cursor: Cursor) {
+        super();
+        this.cursor = cursor;
+    }
+
+    visit(tree: Tree | null, p: PrintOutputCapture<P>): J.J | null {
+        return super.visit(tree, p);
+    }
+
+    visitJsCompilationUnit(cu: JS.CompilationUnit, p: PrintOutputCapture<P>): J.J | null {
```

Comment:
Is there any reason we don't use the `override` keyword? I think that would make the code a bit clearer and also catch any errors when there is no base method corresponding to it.